providers/fab/README.rst

```diff
@@ -23,7 +23,7 @@
 
 Package ``apache-airflow-providers-fab``
 
-Release: ``2.0.0``
+Release: ``1.5.2``
```

Comment:
this isn't right.

Reply:
uh-oh

Reply:
You can't release fab 1.x from main branch

Reply:
what you really want is 2.0.0b